app/models/region.rb

```diff
@@ -31,4 +31,28 @@ def log_payload
     attrs["errors"] = errors.full_messages.join(",") if errors.any?
     attrs.symbolize_keys
   end
+
+  # These methods are generated on class load and updates to RegionTypes
+  # will not be available until the class is reloaded.
+  RegionType.all.map do |region_type|
```

Comment:
Will a surgical `method_missing` be more robust against changing RegionTypes? I know there's a stigma attached to it, like rescuing errors, but I think it's a useful tool when applied properly.

**Update:** On second thought, Region Types should be _super_ stable in any given deployment. As you were, gentlemen.

Reply:
Yeah, considered it. Came to the same conclusion as you.